data/packages/calogica/dbt_date/index.json

```
@@ -3,6 +3,7 @@
     "namespace": "calogica",
     "description": "dbt models for dbt-date",
     "latest": "0.10.1",
+    "redirectname": "godatadriven_dbt_date",
```

Pretty sure this should be a slash shouldn't it?
```suggestion
    "redirectname": "godatadriven/dbt_date",
```

---

This wasn't fully correct as it results in a URL like this: 
```bash
https://hub.getdbt.com/calogica/godatadriven/dbt_date/latest/
```

Instead I added `redirectnamespace` in b36c550 to produce the correct URL.